pkg/client/client.go

```diff
@@ -166,6 +166,68 @@ func (c *ConmonClient) CreateContainer(ctx context.Context, cfg *CreateContainer
 	return response.ContainerPid(), nil
 }
 
+type ExecContainerResult struct {
```

Comment:
client update could even go in another separate commit--but that's up to you. it could also go with the exec commit if you'd like